diff --git a/src/interfaces/ship_interfaces/CMakeLists.txt b/src/interfaces/ship_interfaces/CMakeLists.txt
index 8594bd58b8f47c40bee48db9ad6a738b30989c64..a64033e2b6cfeba6a547074ff644a3ef4f56eaf6 100644
--- a/src/interfaces/ship_interfaces/CMakeLists.txt
+++ b/src/interfaces/ship_interfaces/CMakeLists.txt
@@ -24,6 +24,8 @@ set(msg_files
   "msg/RoutePlan.msg"
   "msg/RoutePlanStatus.msg"
   "msg/AvoidancePlan.msg"
+  "msg/VelocityIntent.msg"
+  "msg/VelocityExecutionStatus.msg"
   "msg/RouteExecutionStatus.msg"
   "msg/NavigationModeStatus.msg"
   "msg/BerthingTask.msg"
